Xamarin.Forms.Platform.Android/Renderers/LabelRenderer.cs

```diff
@@ -23,6 +23,8 @@ public class LabelRenderer : ViewRenderer<Label, TextView>
 		FormsTextView _view;
 		bool _wasFormatted;
 
+		readonly MotionEventHelper _motionEventHelper = new MotionEventHelper();
```

Comment:
The renderer not being colected because MotionEventHelper will keep a reference to it.